src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/components/ScenePresetsPicker/ScenePresetsPicker.tsx

```diff
@@ -176,6 +176,8 @@ export class ScenePresetsPicker extends SceneObjectBase<ScenePresetsPickerState>
     }
 
     if (option.value?.startsWith('auto-select-')) {
+      this.setState({ value: null });
```

Comment:
After applying an "Auto-select" preset, the UI will get back to "Choose a preset":

<img width="264" alt="image" src="https://github.com/user-attachments/assets/f668661e-ce26-47ad-b0ab-ff7e36978974">

Reply:
I wonder if this should be added as a separate method and called in SceneExploreDiffFlameGraph here https://github.com/grafana/explore-profiles/blob/45cea14eef9fbde49f4474051a5833a1be623d2a/src/pages/ProfilesExplorerView/components/SceneExploreDiffFlameGraph/SceneExploreDiffFlameGraph.tsx#L82 🤔